core/slim_sim.cpp

```diff
@@ -4873,34 +4876,75 @@ void SLiMSim::DerivedStatesToAscii(table_collection_t *p_tables)
 	mutation_table_free(&mutations_copy);
 }
 
+void SLiMSim::AddIndividualToTable(Individual *p_individual, table_collection_t *p_tables, bool p_addToRemembered)
+{
+    // Maybe this should take a list of individuals instead of a single one?
```

Comment:
We had to do the same thing in two places, so I factored this out.

Maybe it should take a vector of individuals? I wasn't sure how to get that in one place, though.